testing/src/icon/mock-icon-library.service.ts

```diff
@@ -0,0 +1,15 @@
+import { Injectable } from '@angular/core';
+import { FaIconLibraryInterface } from '@fortawesome/angular-fontawesome';
+import { IconPrefix, IconName, IconDefinition } from '@fortawesome/fontawesome-svg-core';
+import { faCoffee } from '@fortawesome/free-solid-svg-icons';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class MockFaIconLibraryService implements FaIconLibraryInterface {
+  public addIcons() {}
+  public addIconPacks() {}  
+  getIconDefinition(prefix: IconPrefix, name: IconName): IconDefinition {
+    return faCoffee;
```

Comment:
Please return an inline icon here. Consumers of the `@fortawesome/angular-fontawesome` may not have `@fortawesome/free-solid-svg-icons` dependency installed and then this code will fail.

Something like below will render a black square instead of an icon:

```
{
    prefix: 'fad',
    iconName: 'dummy' as IconName,
    icon: [
      512,
      512,
      [],
      'f030',
      [
        'M50 50 H462 V462 H50 Z'
      ]
    ]
  }
```